db/structure.sql

```diff
@@ -10,7 +10,11 @@ CREATE TABLE `attributes` (
   `order` int(11) NOT NULL DEFAULT '1',
   `entity_type_id` int(11) DEFAULT NULL,
   PRIMARY KEY (`id`),
-  KEY `idx_order` (`order`)
+  KEY `idx_order` (`order`),
+  CONSTRAINT fk_attributes_entity_types
+  FOREIGN KEY (entity_type_id)
+  REFERENCES entity_types(id)
+  ON DELETE CASCADE
```

Comment:
I'm not 100% sold on the idea. To me this is a bit scary.

While it is true that the database will be littered with orphan data, on the flip side users might not be aware that deleting an entity type would delete *all* entities of that type. It would be big issue in that scenario.

Reply:
Hello @angeli-mercedes and @Norlant1,

After giving this some thought, I say we do not delete anything implicitly, and we do not delete on cascade. We let the users delete items explicitly.